src/components/DataTable/components/Cell/tests/Cell.test.tsx

```diff
@@ -261,3 +251,14 @@ describe('<Cell />', () => {
     });
   });
 });
+
+function mountWithTable<P>(node: ReactElement) {
+  return mountWithAppProvider<P>(
```

Comment:
Mounting just the `Cell` was invalid html which was causing an error to be thrown